src/views/account-detail/accountDetail.jsx

```diff
@@ -175,19 +173,19 @@ export default function AccountDetail() {
     if (node !== undefined) {
```

Comment:
Instead of wrapping the whole thing in an if, maybe we can just reverse it and return the default state if it's undefined?